engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -895,18 +898,71 @@ namespace dmGameSystem
         return dispatch_context.m_Success;
     }
 
+    static void Step(CollisionWorld* world, PhysicsContext* physics_context, dmGameObject::HCollection collection, const dmPhysics::StepWorldContext* step_ctx)
```

Comment:
The actual step function, with dispatching messages before/after and also updating the gameobject transforms.